api/data_explorer/__main__.py

```diff
@@ -146,6 +146,9 @@ def _get_facets():
 
     for facet_config in facets_config:
         field_name = facet_config['elasticsearch_field_name']
+        if not field_name in mapping['type']['properties']:
+            raise ValueError('Field %s not found in Elasticsearch index %s' %
```

Comment:
nit: `elasticsearch_field_name %s not found...`

Reply:
I think mine is more readable. I think it's obvious we're talking about an Elasticsearch field.

Reply:
For me, if I run docker-compose up and get this error, it's not obvious that "Field" means "one of the elasticsearch_field_names in ui.json config". Being specific reduces obstacles to debugging

Reply:
Done